app/templates/main/search-transferring-body.html

```diff
@@ -2,12 +2,23 @@
     "everything": "Everything",
     "summary": [breadcrumb_values[2]["query"], "Results summary"],
     "body": [breadcrumb_values[0]["transferring_body_id"], breadcrumb_values[1]["transferring_body"]],
-    "search_terms": breadcrumb_values[3]["search_terms"],
 } %}
 {% set standard_user_breadcrumbs = {
     "body": [breadcrumb_values[0]["transferring_body_id"], breadcrumb_values[1]["transferring_body"]],
-    "search_terms": breadcrumb_values[3]["search_terms"],
 } %}
+{% if breadcrumb_values[3]["search_terms"] | length > 0 %}
```

Comment:
what was wrong with the breadcrumbs before?

Reply:
there is no validation on search box at the moment so user can enter space or , in the search box and click on search button to fetch data, it was causing failure if on breadcrumb to show up correctly if there is not value assigned to search term (query)